assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -17,50 +17,131 @@ import useLibrary from './useLibrary';
 
 const Image = styled.img`
 	height: 150px;
-	width: 150px;
-	padding: 4px;
-	margin: 4px;
+	width: 48%;
+	padding: 3px;
+	margin: 3px;
 	border: 1px solid white;
 `;
 
+const Title = styled.h3`
+	color: ${ ( { theme } ) => theme.colors.fg.v1 };
+	margin: 0px;
+	font-size: 19px;
+	line-height: 20px;
+	line-height: 1.4em;
+	flex: 2 0 0;
+`;
+
+const Button = styled.button`
+	 background: #242A3B;
+	 color: #ffffff;
+	 padding: 5px;
+	 font-weight: bold;
+	 flex: 1 0 0;
+	 text-align: center;
+	 border: 0px none;
+`;
+
+const Header = styled.div`
+	display: flex;
+	margin: 0px 4px 10px;
+`;
+
+const Message = styled.div`
+	color: #ffffff;
+	font-size: 19px;
+`;
+
 function MediaLibrary( { onInsert } ) {
 	const {
-		state: { media, isMediaLoading, isMediaLoaded },
-		actions: { loadMedia },
+		state: { media, isMediaLoading, isMediaLoaded, mediaType },
+		actions: { loadMedia, setIsMediaLoading, setIsMediaLoaded },
 	} = useLibrary();
 
-	useEffect( loadMedia );
+	useEffect( () => {
+		loadMedia();
+		// Work around that forces default tab as upload tab.
+		wp.media.controller.Library.prototype.defaults.contentUserSetting = false;
+	} );
+
+	const mediaPicker = () => {
```

Comment:
This should be moved to a independent component. Maybe just a `WPMediaUploadButton`, that also has the visual button.

Reply:
Good point. I have refactored the code so that upload button is a reusable component.  See https://github.com/ampproject/amp-wp/pull/3843/commits/f516d7c1ccab8413c187f81e1bb31a3c37fa4653 
I think we should stir clear of WP naming for now. I think a later piece of work will need to be done to completely abstract this code. For now, I think that `uploadButton` naming should do just fine.